oraclesrv/app.py

```diff
@@ -22,6 +22,9 @@ def create_app(**config):
 
     Discoverer(app)
 
+    with  app.app_context() as ac:
+        docmatch_model()
```

Comment:
I would find it nicer if docmatch_model was returning the model; and app.extension was modified in here - although I admit many extensions do it the way you have it. So maybe rename the function to something like `setup_extension` ?

Reply:
good point. i like it, going to remember to make the same modifications for reference service.